src/click_repl/__init__.py

```diff
@@ -10,20 +10,24 @@
 from .exceptions import InternalCommandException, ExitReplException  # noqa
 
 # Handle backwards compatibility between Click 7.0 and 8.0
```

Comment:
i don't think we need to move this to src

Reply:
ok moving it out as u wish